### ISSUE TITLE
Can't install on .NET Core 3.0

### ISSUE THREAD

**Issue by cosenal** [microsoft/iqsharp#42]
**Describe the bug**
iqsharp can't be installed on .NET Core 3.0 

**To Reproduce**
1. Install .NET Core SDK from https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-3.0.100-macos-x64-installer ✅ 
2. `dotnet build iqsharp.sln` ✅ 
3. `cd src/Tool/` ✅ 
4. `dotnet run -- install` ❌ 
5. See error:
```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.AspNetCore.App', version '2.2.0' was not found.
  - The following frameworks were found:
      3.0.0 at [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The .NET Core frameworks can be found at:
  - https://aka.ms/dotnet-download
```

**Expected behavior**
From the document, I would expect iqsharp to work on
> .NET Core 2.2 or later

Also from this page: https://dotnet.microsoft.com/download/dotnet-core
It looks like  .NET Core 3.0 is the recommended version and 2.2 is reaching end of support soon.

**System information**
```
.NET Core SDK (reflecting any global.json):
 Version:   3.0.100
 Commit:    04339c3a26

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.14
 OS Platform: Darwin
 RID:         osx.10.14-x64
 Base Path:   /usr/local/share/dotnet/sdk/3.0.100/

Host (useful for support):
  Version: 3.0.0
  Commit:  7d57652f33

.NET Core SDKs installed:
  3.0.100 [/usr/local/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.0.0 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.0.0 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
```

**Comment by bromeg** [comment]
Thank you for opening this! We are aware of this issue and working across our repos to migrate to .NET Core 3.0.

**Comment by cgranade** [comment]
As @bromeg mentioned, we are actively working to resolve this problem; in the meantime, you can work around the issue by installing the [.NET Core SDK, version 2.2](https://dotnet.microsoft.com/download/dotnet-core/2.2) rather than version 3.0. Thanks for your patience!

**Comment by beraybentesen** [comment]
@cgranade Any news or ETA ? Thanks.

**Comment by cgranade** [comment]
@beraybentesen: Apologies, we forgot to close this issue when we released 0.10, but it should be fixed now! Please let us know (either by reopening or by filing a new issue) if you still encounter any problems. Thanks!
